### DISCUSSION TITLE
Moving the repository to a new place

### DISCUSSION THREAD

**Discussion by hugolgst** [vimsence/vimsence#41] [Ideas]
@anned20 suggested moving this fork to an organization in order to have more flexibility on the contributions.
It sounds like a great idea.

However I am a bit concerned about the current users using the repository, they might have to change the path for the plugin.
Should we keep a mirror here on my profile?

**Comment by hugolgst** [comment]
Also, can we make the Discord Application an organization?

**Reply by anned20** [reply]
This might be a use case for [Discord Teams](https://discord.com/developers/teams)

**Comment by anned20** [comment]
I think that, as long as you don't create a new repository named the exact same, GitHub will redirect it to the new repository if you move it using the "move" feature.

**Reply by hugolgst** [reply]
I remember that it is only during 30 days, am I right?

**Reply by anned20** [reply]
According to [this](https://github.community/t/how-long-does-github-forward-renamed-moved-repos/582) forum post, it seems the redirect stays in place indefinitely.

**Comment by anned20** [comment]
Shall I pick up the code changes? README and such?

**Reply by hugolgst** [reply]
As you want!
I will begin to move the Discord Application

**Reply by anned20** [reply]
I will send you my Discord username through e-mail ;-)

**Reply by hugolgst** [reply]
I am going to move the application this week!

**Reply by hugolgst** [reply]
Could you send me your Discord username @anned20 ?

**Reply by anned20** [reply]
Sent it @hugolgst.

**Reply by hugolgst** [reply]
Oh your email arrived in spam, sorry I did not see that.

**Comment by hugolgst** [comment]
Thanks, @anned20, for the initiative; I created the organization and moved the repository to it.
I also created two teams, @vimsence/owners and @vimsence/maintainers.

I think we can freely add people that want to contribute to the maintainers' team.
